css/comp.css

```diff
@@ -41,3 +42,253 @@
   height: 12rem;
   width: 12rem;
 }
+
+/*----------------------------------- badges */
+
+.badge {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  border-radius: 50%;
+  border: none;
+  font-size: 1.2em;
+  right: 0.5em;
+  top: 0;
+  width: 2.5em;
+  height: 2.5em;
+}
+
+.icon-containter {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 2rem;
+}
+
+.badge-icon {
+  color: var(--darker);
+  font-size: 10rem !important;
+}
+
+.badge-st {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  border-radius: 50%;
+  border: none;
+  font-size: 1.2em;
+  right: 1em;
+  bottom: 0;
+  width: 1.8em;
+  height: 1.8em;
+}
+
+.badge-primary {
+  background-color: var(--dark-primary);
+  color: white;
+}
+.badge-danger {
+  background-color: var(--danger);
+  color: white;
+}
+
+.badge-warning {
+  background-color: var(--warning);
+  color: var(--darker);
+}
+
+.badge-success {
+  background-color: var(--success);
+  color: var(--darker);
+}
+
+.badge-dark {
+  background-color: var(--darker);
+  color: var(--light);
+}
+
+.badge-default {
+  background-color: var(--light);
+  color: black;
+}
+
+/* ---------------------------------------------- Buttons */
+
+.btn {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 0.5rem;
+  font-size: 2rem;
+  padding: 1rem 1.5rem;
+  justify-content: center;
+  border: none;
+  cursor: pointer;
+  border-radius: 0.5rem;
+  transition: 350ms;
+}
+
+.btn:hover {
+  background-color: var(--light-primary);
+}
+
+.btn-primary {
+  display: inline-flex;
+  justify-content: center;
+  margin: 0 0.5rem;
+  font-size: 2rem;
+  background-color: white;
+  color: var(--dark-primary);
+  border: 0.1rem solid var(--dark-primary);
+  padding: 1rem 1.5rem;
+  justify-content: center;
+  cursor: pointer;
+  border-radius: 0.5rem;
+  transition: 350ms;
+}
+
+.btn-primary:hover {
+  background-color: var(--dark-primary);
+  color: white;
+}
+
+.btn-warning {
+  display: inline-flex;
+  justify-content: center;
+  margin: 0 0.5rem;
+  font-size: 2rem;
+  background-color: white;
+  color: var(--warning);
+  border: 0.1rem solid var(--warning);
+  padding: 1rem 1.5rem;
+  justify-content: center;
+  cursor: pointer;
+  border-radius: 0.5rem;
+  transition: 350ms;
+}
+
+.btn-warning:hover {
+  background-color: var(--warning);
+  color: white;
+}
+
+.btn-danger {
+  display: inline-flex;
+  justify-content: center;
+  align-content: center;
+  margin: 1rem;
+  font-size: 2rem;
```

Comment:
These properties are in the `.btn` class so you can just add them in HTML.
Example-- `<button class="btn btn-danger"></button>`

Reply:
yes initially i did but i wanted to write only one class eg `<class= btn-primary>` instead of `<class = btn btn-primary` so that's why i had to write separate code. but i will do these changes for sure.